### PR TITLE
add tutorials in English and fix some description errors

### PR DESCRIPTION
**Pull request by kuailehaibin**
> + add document Message-handling-with-function-module in English
> + add document Message-synchronize-with-iothub-through-remote-module in English
> + fix some description errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,43 +30,43 @@ More detailed documents of design are as follows:
 
 ## Concepts
 
-OpenEdge is made up of **main program module, Local Hub Module, Local Function Module, MQTT Remote Module and Python2.7 Runtime Module.** The main capabilities of each module are as follows:
+OpenEdge is made up of **Main Program Module, Local Hub Module, Local Function Module, MQTT Remote Module and Python2.7 Runtime Module.** The main capabilities of each module are as follows:
```
Comment (ludanfeng) [inline README.md]: Main Program Module -> Master Program
主程序不是模块
全局改一下，你没看我的翻译或者设计图吗?

Comment (kuailehaibin) [reply README.md]: 好的

File: README.md [hunk 2]
```diff
@@ -30,9 +30,9 @@ More detailed documents of design are as follows:
...
+OpenEdge is made up of **Master Program, Local Hub Module, Local Function Module, MQTT Remote Module and Python2.7 Runtime Module.** The main capabilities of each module are as follows:
 
-> + **Main Program** is used to manage all modules's behavior, such as start, stop, etc. And it is composed of module engine, API and cloud agent.
+> + **Master Program** is used to manage all modules's behavior, such as start, stop, etc. And it is composed of module engine, API and cloud agent.
 > + **Module Engine** controls the behavior of all modules, such as start, stop, restart, listen, etc, and currently supports **Docker Container Mode** and **Native Process Mode**.
 > + **Cloud Agent** is responsible for the communication with **Cloud Management Suite** of [BIE](https://cloud.baidu.com/product/bie.html), and supports MQTT and HTTPS protocols. In addition, if you use MQTT protocol for communication, **MUST** take two-way authentication of SSL/TLS; otherwise, you **MUST** take one-way authentication of SSL/TLS due to HTTPS protocol.
 > + The main program exposes a set of **HTTP API**, which currently supports to start, stop and restart module, also can get free port.
```
Comment (ludanfeng) [inline README.md]: main -> master

File: doc/us-en/overview/OpenEdge-concepts.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+OpenEdge is made up of **Master Program, Local Hub Module, Local Function Module, MQTT Remote Module and Python Runtime Module.** The main capabilities of each module are as follows:
 
-> + **Main Program** is used to manage all modules's behavior, such as start, stop, etc. And it is composed of module engine, API and cloud agent.
+> + **Master Program** is used to manage all modules's behavior, such as start, stop, etc. And it is composed of module engine, API and cloud agent.
 >   + **Module Engine** controls the behavior of all modules, such as start, stop, restart, listen, etc, and currently supports **Docker Container Mode** and **Native Process Mode**.
 >   + **Cloud Agent** is responsible for the communication with **Cloud Management Suite** of [BIE](https://cloud.baidu.com/product/bie.html), and supports MQTT and HTTPS protocols. In addition, if you use MQTT protocol for communication, **MUST** take two-way authentication of SSL/TLS; otherwise, you **MUST** take one-way authentication of SSL/TLS due to HTTPS protocol.
 >   + The main program exposes a set of **HTTP API**, which currently supports to start, stop and restart module, also can get free port.
```
Comment (ludanfeng) [inline doc/us-en/overview/OpenEdge-concepts.md]: +1

File: README.md (hunk 2 above)
Comment (kuailehaibin) [reply README.md]: 好的

**Pull request merged**